src/components/Explore/Explore.tsx

```diff
@@ -80,14 +80,15 @@ function getLabelLength(series: Series, shortLabel: boolean) {
 }
 
 const Explore: React.FunctionComponent<{
-  fips: string;
-}> = ({ fips }) => {
+  fipsList: string[];
```

Comment:
initialFipsList?
Am I understanding it right that this what the explore module is intialized to before the user picks whatever locations they want?

Reply:
Correct. The parent component (`LocationPage` or `Homepage`) determines the initial set of locations for the component and then the user can add more or remove them. For the location page, the initial location corresponds to the page location, for the home page, is a random set of states.